docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,92 @@
+
+![Click behavior icon](images/interactive-dashboards/click-behavior-icon.png)
+
+Metabase will slide out the **Click behavior sidebar**. For questions composed using the query builder, you can select from 3 options:
+
+- Open the Metabase action menu.
+- Go to a custom destination.
+- Update a dashboard filter (if the dashboard has a filter)
+
+SQL questions will only have the option to **Go to a custom destination**, and **Update a dashboard filter**, as the action menu (which allows users to drill through the data) is only available to questions composed with the query builder.
+
+If your dashboard has a filter, you'll also see an option to update the filter (see [cross-filtering](#cross---filtering) below).
+
+![Click behavior menu](images/interactive-dashboards/click-behavior-menu.png)
+
+### Open the action menu
+
+For questions composed using the query builder, the default click behavior is to open the **action menu**, which presents people with the option to [drill through the data](https://www.metabase.com/blog/drilling-through-data/index.html):
+
+![Action menu](images/interactive-dashboards/action-menu.png)
+
+### Custom destinations
+
+You can set custom destinations for all questions, including SQL questions. 
+
+![Custom destinations](images/interactive-dashboards/custom-destination.png)
+
+Possible destinations include:
+
+- Dashboards
+- Saved questions
+- URLs
+
+You can also pass a value to that destination from the clicked card. For example, if you link to a dashboard that has a filter for category, you can pass a value for category from the origin question to the destination dashboard:
+
+![Pass value to dashboard](images/interactive-dashboards/pass-value.png)
+
+Once you select a column, Metabase will display the column used to pass the value, as well as the filter the value will be passed to at the destination dashboard.
+
+![Pass category to filter](images/interactive-dashboards/pass-category-to-filter.png)
+
+In the example above, when a user clicks on the **Orders by product category** card, Metabase will pass the clicked category to the destination dashboard, which will then filter its cards by that category.
+
+When displaying questions as tables, you can select different click behaviors for different columns in the table.
+
+You can also use values to construct URLs to external resources.
+
+![Enter a URL](images/interactive-dashboards/enter-a-url.png)
+
+From the **Click behavior** sidebar, select **Go to a custom destination** and link to **URL**. The **Enter a URL to link to** modal will pop up, allowing you to specify a URL, as well as specify the value of a column or dashboard filter. Click on the dropdown menu **Values you can reference** to see your options.
+
+### Cross-filtering a dashboard
+
+If your dashboard contains a filter, you can use a chart to update the value supplied to that filter. 
```

Comment:
```suggestion
If your dashboard contains at least one filter, you can set things up so that clicking on a chart in the dashboard will update a filter. 
```